### DISCUSSION TITLE
Can I run the emulator on my Linux Debian Laptop?

### DISCUSSION THREAD

**Discussion by ghost** [Berny23/LD-ToyPad-Emulator#146] [Q&A]
Can I run the emulator on my Linux Debian Laptop? I do not have access to an SBC but I do have a Linux Debian laptop which was originally a Windows laptop but with Linux replacing it. I want to be able to run Lego Dimensions Toy Pad Emulator on my ps3 Slim Model and using the Linux Laptop to replace what is supposed to be the SBC. Is this possible? I have access to flash drives if it is needed.

**Comment by DiverseBloom780** [comment]
> Can I run the emulator on my Linux Debian Laptop? I do not have access to an SBC but I do have a Linux Debian laptop which was originally a Windows laptop but with Linux replacing it. I want to be able to run Lego Dimensions Toy Pad Emulator on my ps3 Slim Model and using the Linux Laptop to replace what is supposed to be the SBC. Is this possible? I have access to flash drives if it is needed.

you should be able to do that, i got the ToyPad running on Ubuntu but not sure what USB client to use on Linux cause the Virtual here USB client is too confusing to setup, wish there was a better one for Linux, but i know for PS3 you have to have the Pi Zero Plugged into a power supply even tho it will be connected to the PS3, not sure if SBC is required tho
